content/ch-gates/oracles.ipynb

```diff
@@ -0,0 +1,289 @@
+      "text/plain": [
+       "<Figure size 423.808x204.68 with 1 Axes>"
+      ]
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qc = Uf + Vf.inverse()\n",
+    "\n",
+    "qc.draw(output='mpl',justify='none')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This circuit begins with two identical `cx` gates, whose effects cancel each other out. All that remains is the final `cx` between the input and garbage registers. Mathematically, this means\n",
+    "\n",
+    "$$\n",
+    "V_f^\\dagger U_f \\left| x,f(x),0 \\right\\rangle = \\left| x , 0 ,g(x) \\right\\rangle.\n",
```

Comment:
the equation should be
"V_f^\\dagger U_f \\left| x, 0, 0 \\right\\rangle = \\left| x , 0 ,g(x) \\right\\rangle.\n" 
I think...